can/broadcastmanager.py

```diff
@@ -207,13 +207,28 @@ def __init__(
         messages: Union[Sequence[Message], Message],
         period: float,
         duration: Optional[float] = None,
+        on_error: Optional[Callable[[Exception], bool]] = None,
     ):
+        """Transmits `messages` with a `period` seconds for `duration` seconds on a `bus`.
+
+        The `on_error` is called if any error happens on `bus` while sending `messages`.
+        If `on_error` present, and returns ``True`` when invoked, thread is
```

Comment:
We will probably add the same functionality to the Notifier as well, but to keep backwards compatibility it will probably be implemented the other way around, i.e. return True to keep thread alive, otherwise it will stop like today. So to make the behavior consistent I think you should switch too.

Reply:
Done